Analytics/resources/moving_sensors/get_location_data.py

```diff
@@ -0,0 +1,48 @@
+from datetime import datetime
+from http import HTTPStatus
+
+from flask_restful import Resource
+from flask_restful import reqparse
+
+from models.pin_location_data import LocationData, Tracker
+
+
+class GetLocationData(Resource):
+    """
+    Fetch Location Data
+    """
+
+    def __init__(self) -> None:
+        """
+        Set required arguments for GET request
+        """
+        self.reqparser = reqparse.RequestParser()
+        self.reqparser.add_argument('tracker_id', required=False, store_missing=False, type=str)
+        self.reqparser.add_argument('start_date', required=False, default=datetime.now(),
+                                    help='Datetime Format: %d/%m/%y',
```

Comment:
Double check this is correct. `%d/%m/%y` is 25/01/19 while `%d/%m/%Y` is 25/01/2019. You use `%d/%m/%Y`else where